src/Uno.UWP/Media/Playback/MediaPlayer.cs

```diff
@@ -58,7 +58,7 @@ public double Volume
 			get => _volume;
 			set
 			{
-				_volume = value;
+				_volume = Math.Clamp(value, 0.0, 1.0);
```

Comment:
From WinUI's docs: "The audio volume for media playback. The allowed range of values is 0 to 1. Values outside of this range will be ignored.". So, I would instead ignore instead of clamping.

Reply:
Right, I will change this.

Reply:
Done